setup.py

```diff
@@ -9,23 +16,37 @@
     author=u'Piotr Szymański',
     author_email='niedakh@gmail.com',
     license='BSD',
-#    long_description=open('README.md').read(),
+    #    long_description=open('README.md').read(),
     url='http://scikit-multilearn.github.io/',
-    description= 'A set of python modules for multi-label classification',
+    description='A set of python modules for multi-label classification',
     classifiers=[
-          'Development Status :: 4 - Beta',
-          'Environment :: Console',
-          'Environment :: Web Environment',
-          'Intended Audience :: Developers',
-          'Intended Audience :: Education',
-          'Intended Audience :: Science/Research',
-          'License :: OSI Approved :: BSD License',
-          'Operating System :: MacOS :: MacOS X',
-          'Operating System :: Microsoft :: Windows',
-          'Operating System :: POSIX',
-          'Programming Language :: Python',
-          'Topic :: Scientific/Engineering',
-          'Topic :: Scientific/Engineering :: Information Analysis',
-          'Topic :: Scientific/Engineering :: Bio-Informatics',
-          ],
-)
\ No newline at end of file
+        'Development Status :: 4 - Beta',
+        'Environment :: Console',
+        'Environment :: Web Environment',
+        'Intended Audience :: Developers',
+        'Intended Audience :: Education',
+        'Intended Audience :: Science/Research',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: MacOS :: MacOS X',
+        'Operating System :: Microsoft :: Windows',
+        'Operating System :: POSIX',
+        'Programming Language :: Python',
+        'Topic :: Scientific/Engineering',
+        'Topic :: Scientific/Engineering :: Information Analysis',
+        'Topic :: Scientific/Engineering :: Bio-Informatics',
+    ],
+    ext_modules=[
+        Extension("skmultilearn.tree.pctcriterium",
+                  ["skmultilearn/tree/pctcriterium.c"],
+                  extra_compile_args=["-Zi", "/Od"],
```

Comment:
These compile args are compatible only with the MSVC compiler on Windows; there should be a switch to remove/replace them on Linux/Mac platforms

Reply:
requirements.txt also needs to be cleaned up from the existing python modules to allow installation of this package